accepted/2023/owner-author-pmui.md

```diff
@@ -0,0 +1,214 @@
+# Display Trusted Owners on the PM UI Browse Tab
```

Comment:
Is `Trusted` too strong here? Who trusts these owners? How can I declare that I trust someone?

Should we just call them owners?

Reply:
Another word can work. The problem with calling everything "owners" is in the motivation; the word has multiple meanings and doesn't convey whether it's from a source of trust (package source) or a piece of metadata in the package.

Reply:
> from a source of trust

Similar as earlier, `trust` raises too many questions :D 

`Package source owners`, `Current owners`? 

Words are difficult :D

Reply:
`Package source owners` - I like except that it's a little confusing as it sounds like the admins of the servers.
This isn't too time critical to this spec as it's just the nomenclature we're using in code and eventually docs. Getting a definition/word nailed down sooner than later would be good.

Reply:
> This isn't too time critical to this spec

100%, this is spec/docs feedback, but doesn't change the overall intent of the spec.